draft-ietf-quic-qpack.md

```diff
@@ -650,32 +656,31 @@ entries which are frequently referenced, both to avoid the need to resend the
 header and to avoid the entry in the table blocking the ability to insert new
 headers.
 
-### Dynamic Table Size Update {#size-update}
+### Dynamic Table Capacity Update {#capacity-update}
```

Comment:
```suggestion
### Set Dynamic Table Capacity {#set-capacity}
```

Is this a better formulation?  Set is more direct.

Note that there are more places to fix than this, but you get the idea.